server/routes/subscription.js

```diff
@@ -281,6 +281,9 @@ router.postAsync('/:cid/subscribe', passport.parseForm, corsOrCsrfProtection, as
 
     if (existingSubscription && existingSubscription.status === SubscriptionStatus.SUBSCRIBED) {
         await mailHelpers.sendAlreadySubscribed(req.locale, list, email, existingSubscription);
+        if (req.xhr) {
```

Comment:
I'm afraid this should not be here. If I understand correctly, It essentially allows anyone to check if an email is in the mailing list.

Reply:
Hmm. Yeah you're right on that, good call. We have to return some sort of JSON though or it breaks the widget. Can we just return a success then? That way through the API you won't really know.
```
return res.status(200).json({
    msg: tUI('pleaseConfirmSubscription', req.locale)
});
```